client/main/notifications.js

```diff
@@ -1,19 +1,36 @@
 
 'use strict';
 
-// const Notification = require('./notification');
+var $ = require('jquery');
+
+const NotificationView = require('./notification');
 
 const Notifications = function($el) {
     this.$el = $el;
-    this._requests = [ ];
+    this._list = [ ];
 };
 
 Notifications.prototype.notify = function(request) {
 
     console.log('notification received');
```

Comment:
OK, i think we're almost there! can you drop out these `console.log`s? i just put them in as placemarkers.